packages/core/src/components/TickerList/TickerList.js

```diff
@@ -38,6 +38,7 @@ export const TickerList = (props) => {
   const keyForId = getMappedKey(KEYS.ID, rowMapping)
   const keyForBaseCcy = getMappedKey(KEYS.BASE_CCY, rowMapping)
   const keyForQuoteCcy = getMappedKey(KEYS.QUOTE_CCY, rowMapping)
+  const keyForLabels = getMappedKey(KEYS.LABELS, rowMapping)
```

Comment:
- rename labels this to `baseCcyFullName`
- change matches code to
```
  const matches = _toLower(_join([
        baseCcy,
        quoteCcy,
        baseCcy + quoteCcy,
        baseCcyFullName,
      ]))
```
- also add this new support to TickerList.container, by passing baseCcyFullName in `data` prop,
- after above changes, I think we don't need backward compatibility line